tests/test_database.py

```diff
@@ -1239,6 +1239,34 @@ def test_the_homer_encode_decode_choices(self):
             decoded = db.decode_message('Gear', encoded)
             self.assertEqual(decoded, decoded_message)
 
+    def test_the_homer_encode_decode_choice_scaling(self):
+        """
+        Verify a label/enum matches the raw value, and is not affected by scaling
+        """
+
+        filename = os.path.join('tests', 'files', 'the_homer.kcd')
+        db = cantools.database.load_file(filename)
+        
+
+        messages = [
+            (    0x900,     {'EnumTest': 'one'}, b'\x80\x00\x00\x00\x00\x00\x00\x00'),
+            (    0x900,     {'EnumTest': 'two'}, b'\xFF\x00\x00\x00\x00\x00\x00\x00'),
+            (    0x901,     {'EnumTestFloat': 'one'}, b'\x80\x00\x00\x00\x00\x00\x00\x00'),
+            (    0x901,     {'EnumTestFloat': 'two'}, b'\xFF\x00\x00\x00\x00\x00\x00\x00'),
+
+            #verify encode/decode using int/float to verify scaling still works
```

Comment:
Write as `# Verify encode/decode using int/float to verify scaling still works.`.